css/cart.css

```diff
@@ -0,0 +1,268 @@
+.navbar {
+  background: var(--primary-color);
+  position: static;
+}
+
+strong {
+  font-weight: 900;
+}
+
+.wrap {
+  font-family: sans-serif;
+  width: 85%;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 3rem 0;
+}
+
+.btn {
+  display: inline-block;
+  background: var(--primary-color);
+  color: var(--clr-grey-10);
```

Comment:
Naming shouldn't be name of the color, rather it should be for what purpose we are using.

Reply:
I agree, but the grey color that I am using has different shades, and keeping a track of the shades is feasible this way. I have used that naming convention in my components library where I have three sections, Darker shades of grey, medium dark, and light grey. The numbers denote where it lies on this scale. The number ranges from 900 to 10, 10 being the lightest and 900 being the darkest.

I have opted for this method simply to have a variety and use it accordingly.

But I will certainly make changes to the color variables to better suit this project. Thank you @sainath7878 for reviewing.

Reply:
I have updated the color variables, please have a look at it.